mysql/README.md

```diff
@@ -1,41 +1,88 @@
-dockerfiles-fedora-MySQL
-========================
+MySQL for general usage and OpenShift - Docker image
+======================================================
 
-This repo contains a recipe for making Docker container for SSH and MySQL on Fedora. 
+This repository contains Dockerfiles for MySQL images for general usage and for OpenShift.
 
```

Comment:
Dockerfile without the 's' here, I think. Technically this directory isn't a repository of its own either.

Maybe "This directory contains a Dockerfile to build MySQL images..."